javascript/ql/src/Security/CWE-079/Xss.ql

```diff
@@ -15,8 +15,13 @@ import javascript
 import semmle.javascript.security.dataflow.DomBasedXss::DomBasedXss
 import DataFlow::PathGraph
 
-from Configuration cfg, DataFlow::PathNode source, DataFlow::PathNode sink
-where cfg.hasFlowPath(source, sink)
+from DataFlow::Configuration cfg, DataFlow::PathNode source, DataFlow::PathNode sink
+where
+  (
+    cfg instanceof HtmlInjectionConfiguration or
```

Comment:
In these BDD-times, could you perhaps also check if the compiler stays happy if you include a few more (dummy) configurations in this disjunct?

Reply:
Adding two more configs seems to trigger the memory limit 😬